api_tests/preprint_providers/views/test_preprint_provider_preprints_list.py

```diff
@@ -1,9 +1,8 @@
 from nose.tools import *  # flake8: noqa
```

Comment:
Note for reviewers: I know we are switching to pytest, but all these other tests were still in nose format, so I wanted to keep it consistent. I imagine switching all of them will be easier than having mixed files